osc-server/src/main/java/org/osc/core/broker/service/securitygroup/BindSecurityGroupService.java

```diff
@@ -70,7 +71,7 @@
 	private ConformService conformService;
 
 	@Reference
-	private ApiFactoryService apiFactoryService;
+	public ApiFactoryService apiFactoryService;
```

Comment:
can be package private

Reply:
This not visible in unit test case to mock ApifactoryServices. Hence changed to public.